frontend/public/components/dashboard/dashboards-page/overview-dashboard/inventory-card.tsx

```diff
@@ -68,46 +68,35 @@ const getFirehoseResource = (model: K8sKind) => ({
 const ClusterInventoryItem = withDashboardResources<ClusterInventoryItemProps>(
   React.memo(
     ({
-      watchK8sResource,
-      stopWatchK8sResource,
-      resources,
       model,
       mapper,
       useAbbr,
       additionalResources,
       expandedComponent,
-    }) => {
-      React.useEffect(() => {
-        const resource = getFirehoseResource(model);
-        watchK8sResource(resource);
-        if (additionalResources) {
-          additionalResources.forEach(watchK8sResource);
-        }
-        return () => {
-          stopWatchK8sResource(resource);
-          if (additionalResources) {
-            additionalResources.forEach(stopWatchK8sResource);
-          }
-        };
-      }, [watchK8sResource, stopWatchK8sResource, model, additionalResources]);
+    }: ClusterInventoryItemProps) => {
+      const allResources = React.useMemo(
+        () => [getFirehoseResource(model), ...(additionalResources || [])],
```

Comment:
This is exactly the flexibility I've meant in my comment above :smiley:

```ts
[getFooResource(), ...extraResources]
```

is more meaningful than

```ts
[...getFooResource(), ...extraResources]
```